csharp/src/Apache.Arrow/Ipc/ArrowReaderImplementation.cs

```diff
@@ -158,20 +162,21 @@ internal static ByteBuffer CreateByteBuffer(ReadOnlyMemory<byte> buffer)
             }
 
             var arrowBuff = new[] { nullArrowBuffer, valueArrowBuffer };
+            var offspring = GetOffspring(recordBatchManipulator, field, bodyData);
```

Comment:
I named it "offspring" because the method actually gets all descendants. However, as you said, it should have been called "children" according to the spec.